src/components/atoms/Input/Input.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { InputWrapper, Label, StyledIcon, StyledInput } from './Input.styled';
+import { IProps } from './Input.model';
+
+const Input = ({ type = 'text', icon: Icon, label, ...props }: IProps): JSX.Element => {
+  const renderLabel = () => (label ? <Label>{label}</Label> : null);
```

Comment:
We dont need to use null, better use logical AND operator :)

```suggestion
  const renderLabel = () => (label && <Label>{label}</Label>);
```